CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to lsp6-signer.js
+
+## Types / Typechain
+
+To update the types generated by [Typechain](https://github.com/ethereum-ts/TypeChain) run the following `npm` command:
+
+```bash
+npm run generate-types
```

Comment:
We can delete this section about generating the types, we don't use them in this lib